src/cswinrt/code_writers.h

```diff
@@ -3767,18 +3772,29 @@ evt.Name());
     {
         w.write(R"(
 private volatile static global::System.Runtime.CompilerServices.ConditionalWeakTable<object, EventSource<%>> _%_;
-private static global::System.Runtime.CompilerServices.ConditionalWeakTable<object, EventSource<%>> Make%Table()
+private static global::System.Runtime.CompilerServices.ConditionalWeakTable<object, EventSource<%>> _%
 {
-    %
-    global::System.Threading.Interlocked.CompareExchange(ref _%_, new(), null);
-    return _%_;
+    get
+    {
+        var table = _%_;
+        if (table != null)
```

Comment:
Use `is not null` in all these comparisons, or it'll actually call the overloaded operator